src/app/courses/courses.component.ts

```diff
@@ -214,12 +216,20 @@ export class CoursesComponent implements OnInit, AfterViewInit, OnDestroy {
     this.parent ? this.router.navigate([ '/manager' ]) : this.router.navigate([ '/' ]);
   }
 
-  enrollInSelected(courseIds, type) {
+  enrollLeaveToggle(courseIds, type) {
     this.coursesService.courseAdmissionMany(courseIds, type).subscribe((res) => {
       this.selectedNotEnrolled = 0;
+      this.selectedNotLeft = 0;
```

Comment:
By setting both counts to 0. both the Join & Leave buttons end up disabled regardless of which you click.  If you click one, the other should match the total number selected (i.e. clicking Join would mean all selected courses can be left).